scripts/jenkins/master.sh

```diff
@@ -17,14 +17,7 @@ function init {
 
 function build {
   cd ../..
-
-  if [[ $NO_TEST == "true" ]]; then
```

Comment:
If the `--no-test` option is not used any more, should we remove it from `ARG_DEFS` [above][1] ?

[1]: https://github.com/angular/angular.js/pull/13568/files#diff-8ba5664b63cf24ac7a492e66ec3364fcR7